casual/casual-api/src/main/java/se/laz/casual/config/json/EventServer.java

```diff
@@ -3,36 +3,36 @@
  *
  * This software is licensed under the MIT license, https://opensource.org/licenses/MIT
  */
-package se.laz.casual.config;
+package se.laz.casual.config.json;
 
 import java.util.Objects;
 
-public final class EventServer
+final class EventServer
 {
-    private final int portNumber;
-    private final boolean useEpoll;
+    private final Integer portNumber;
```

Comment:
This is a json pojo only, if it is not present I do not want any kind of defaulting within the json pojo either in the builder as this is not invoked by gson or through primatives as i want final fields and as i want all defaults controlled by the ConfigurationDefaults class only.
This was part of the issue when refactoring, there were lots of different locations where the default could be determined. I have now consolidated that into one place. As a result we use Objects rather than primatives, but only in the package private json POJOs so that we can handle these scenarios and get the correct default values set when they are in fact missing in the json i.e. null.